frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckUserExistsHandler.java

```diff
@@ -184,12 +188,27 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     AuditService.UNKNOWN,
                     persistentSessionId,
                     pair("rpPairwiseId", rpPairwiseId));
+
+            ArrayList<LockoutInformation> lockoutInformation = new ArrayList<>();
+            for (JourneyType journeyType :
+                    new JourneyType[] {JourneyType.SIGN_IN, JourneyType.PASSWORD_RESET}) {
+                long lockTTL =
+                        codeStorageService.getMfaCodeBlockTimeToLive(
+                                emailAddress, MFAMethodType.AUTH_APP, JourneyType.SIGN_IN);
```

Comment:
this is a very specific lock to retrieve. should we not be looking for any and all locks?

Reply:
Yes, sorry, the aim of this ticket was only auth app in sign-in and reset-password, updated the Jira ticket